src/client/hud.cpp

```diff
@@ -735,11 +735,12 @@ void drawItemStack(video::IVideoDriver *driver,
 		driver->draw2DRectangle(color, progressrect2, clip);
 	}
 
-	if(font != NULL && item.count >= 2)
+	const std::string &count_text = item.metadata.getString("count", 0);
+	if((font != NULL && item.count >= 2) || !count_text.empty())
```

Comment:
`if ((!font &&` ...

Reply:
I guess you meant `if ((font &&...` (without the `!`).
I've corrected this line.